docs/UserGuide.md

```diff
@@ -56,15 +65,32 @@ Format: `list`
 
 ### Listing all opening outcomes : `outcome`
 
-Shows the list of openings based on the opening status provided. 
+Shows the list of openings based on the opening status provided.
 
-Format: `outcome /o STATUS`
+Format: `outcome o/STATUS`
 
 * Shows all openings with specified `STATUS`.
-* The status **must be of either ongoing, successful or rejected**.
+* `STATUS` **must be of either ongoing, successful or rejected**.
```

Comment:
Perhaps change terminology of successful to accepted, to btr reflect status (opposite of rejected, and in past tense)

Reply:
Ongoing is also quite vague